chains/solana/contracts/programs/ccip-router/src/event.rs

```diff
@@ -119,6 +119,18 @@ pub mod events {
             pub from: Pubkey,
             pub to: Pubkey,
         }
+
+        #[event]
+        pub struct AddedAllowedOfframp {
+            pub source_chain_selector: u64,
+            pub offramp: Pubkey,
+        }
+
+        #[event]
+        pub struct RemovedAllowedOfframp {
+            pub source_chain_selector: u64,
+            pub offramp: Pubkey,
+        }
```

Comment:
Nice that you already added events :+1:
I've looked at the [EVM implmentation](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/ccip/Router.sol#L34-L35) which has similar events, would you mind renaming these to match EVM?